### DISCUSSION TITLE
macOS support?

### DISCUSSION THREAD

**Discussion by marty-wang** [Shopify/flash-list#503] [Ideas]
Given the similarity of ios and macOS, how do you think adding support for macOS? We are building desktop apps for both Mac and Windows. It would be great that we replace flatlist with flashlist!

**Comment by naqvitalha** [comment]
We don't have plans to write native components for macOS but we're working on making it work with just JS on unsupported platforms.

**Comment by naqvitalha** [comment]
v1.2.0 has been released with macOS/Windows support.
